lib/hacluster.pm

```diff
@@ -646,9 +646,10 @@ sub ha_export_logs {
     upload_logs($mdadm_conf, failok => 1);
 
     # supportconfig
-    my $ret = script_run "supportconfig -g -B $clustername", 300, die_on_timeout => 0;
+    enter_cmd "supportconfig -g -B $clustername; echo DONE";
```

Comment:
why this one does not have `> /dev/$serialdev` ?

Reply:
'cause I forgot it :) Fixed